src/main/java/school/hei/haapi/authentication/JwtConfiguration.java

```diff
@@ -0,0 +1,56 @@
+package school.hei.haapi.authentication;
+
+import java.io.Serializable;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JwtConfiguration implements Serializable {
+  private static final String USER_POOL_ID = ("us-east-1_AvlIh6liI");
```

Comment:
NOT in hard-coded like this. You should use a `final String userPoolId` which you will define in your `application-ryan.properties`. Then you launch the app with `ryan` profile

Reply:
I was stuck here for hours.
Finally I just set environment variables in the JVM Options before the launch but even if the profile 'ryan' is active, I think I don't use properly the configuration ...